logstash-filter-elasticsearch.gemspec

```diff
@@ -24,7 +24,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'elasticsearch', ">= 7.14.9" # LS >= 6.7 and < 7.14 all used version 5.0.5
   s.add_runtime_dependency 'manticore', ">= 0.7.1"
   s.add_runtime_dependency 'logstash-mixin-ca_trusted_fingerprint_support', '~> 1.0'
-  s.add_runtime_dependency 'logstash-mixin-normalize_config_support', '~>1.0'
```

Comment:
This is still an active dependency in this branch

Reply:
Oops, indeed!
I have put it back with https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/190/commits/d72f5a8b74a6953c7be48cca267fbd58881920f8 commit